docs/hadoop-catalog.md

```diff
@@ -63,7 +79,12 @@ Refer to [Schema operation](./manage-fileset-metadata-using-gravitino.md#schema-
 
 ### Fileset properties
 
-None.
+| Property name                                      | Description                                                                                            | Default value                | Required                                                                                   | Since Version   |
+|----------------------------------------------------|--------------------------------------------------------------------------------------------------------|------------------------------|--------------------------------------------------------------------------------------------|-----------------|
+| `authentication.impersonation-enable`              | Whether to enable impersonation for the Hadoop catalog fileset.                                        | The parent's (catalog) value | No                                                                                         | 0.6.0           |
+| `authentication.type`                              | The type of authentication for Hadoop catalog fileset, currently we only support `kerberos`, `simple`. | The parent's (catalog) value | No                                                                                         | 0.6.0           |
+| `authentication.kerberos.principal`                | The principal of the Kerberos authentication for the fileset.                                          | The parent's (catalog) value | required if the value of `authentication.type` is Kerberos and parent hasn't set the value | 0.6.0           |
```

Comment:
This value isn't required. You can fill no directly.

Reply:
OK, I see, if users do not specify the value, the default value is `UserGroupInformation.getCurrentUser`

Reply:
I mean that every authentication config options aren't required. Because they are extend from their parents. Only the oldest parent is required.